### PR TITLE
[PowerPoint] (APIs) Shapes APIs are released as req set 1.4

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

File: docs/requirement-sets/powerpoint/powerpoint-preview-apis.md [hunk 1]
```diff
@@ -16,8 +16,7 @@ The first table provides a concise summary of the APIs, while the subsequent tab
 
 | Feature area | Description | Relevant objects |
 |:--- |:--- |:--- |
-| Slide management | Adds support for adding slides as well as managing slide layouts and slide masters. | [Slide](/javascript/api/powerpoint/powerpoint.slide)<br>[SlideLayout](/javascript/api/powerpoint/powerpoint.slidelayout)<br>[SlideMaster](/javascript/api/powerpoint/powerpoint.slidemaster)|
-| Shapes | Adds support for getting references to the shapes in a slide. | [Shape](/javascript/api/powerpoint/powerpoint.shape) |
+| Selection APIs | Adds more ways to programmatically select slides, shapes, and text ranges. | See the table later in this article. |
 
 ## API list
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/powerpoint/powerpoint-preview-apis.md:22]: There's a typo in the next sentence (on line 23) that mentions Excel. It should say PowerPoint

File: generate-docs/GenerateDocs.cmd [hunk 2]
```diff
@@ -70,6 +70,7 @@ call node version-remover ..\api-extractor-inputs-outlook-release\outlook_1_2\ou
 call node ..\scripts\versioned-dts-cleanup ..\api-extractor-inputs-outlook-release\outlook_1_1\outlook.d.ts Outlook 1.1
 call node version-remover ..\api-extractor-inputs-outlook-release\outlook_1_1\outlook.d.ts "Mailbox 1.1" .\tool-inputs\outlook-base.d.ts
 
+call node version-remover ..\api-extractor-inputs-powerpoint-release\powerpoint_1_3\powerpoint.d.ts "PowerPointApi 1.4" ..\api-extractor-inputs-powerpoint-release\powerpoint_1_3\powerpoint.d.ts
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/GenerateDocs.cmd:73]:
```suggestion
call node version-remover ..\api-extractor-inputs-powerpoint-release\powerpoint_1_4\powerpoint.d.ts "PowerPointApi 1.4" ..\api-extractor-inputs-powerpoint-release\powerpoint_1_3\powerpoint.d.ts
```

File: docs/includes/powerpoint-preview.md [hunk 3]
```diff
@@ -1,52 +1,16 @@
...
 ||[getSelectedTextRangeOrNullObject()](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-getselectedtextrangeornullobject-member(1))|Returns the selected {@link PowerPoint.TextRange} in the current view of the presentation.|
 ||[setSelectedSlides(slideIds: string[])](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-setselectedslides-member(1))|Selects the slides in the current view of the presentation.|
-|[Shape](/javascript/api/powerpoint/powerpoint.shape)|[fill](/javascript/api/powerpoint/powerpoint.shape#powerpoint-powerpoint-shape-fill-member)|Returns the fill formatting of this shape.|
-||[getParentSlide()](/javascript/api/powerpoint/powerpoint.shape#powerpoint-powerpoint-shape-getparentslide-member(1))|Returns the parent {@link PowerPoint.Slide} object that holds this `Shape`.|
+|[Shape](/javascript/api/powerpoint/powerpoint.shape)|[getParentSlide()](/javascript/api/powerpoint/powerpoint.shape#powerpoint-powerpoint-shape-getparentslide-member(1))|Returns the parent {@link PowerPoint.Slide} object that holds this `Shape`.|
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/powerpoint-preview.md:8]: doesn't seem like the @ links didn't resolve

File: docs/docs-ref-autogen/powerpoint/powerpoint/powerpoint.interfaces.presentationloadoptions.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@ uid: 'powerpoint!PowerPoint.Interfaces.PresentationLoadOptions:interface'
 package: powerpoint!
 fullName: PowerPoint.Interfaces.PresentationLoadOptions
 summary: ''
-remarks: '\[ [API set: PowerPointApi 1.1](/javascript/api/requirement-sets/powerpoint/powerpoint-api-requirement-sets) \]'
+remarks: '\[ [API set: PowerPointApi 1.0](/javascript/api/requirement-sets/powerpoint/powerpoint-api-requirement-sets) \]'
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/docs-ref-autogen/powerpoint/powerpoint/powerpoint.interfaces.presentationloadoptions.yml:7]: Good catch. This should not have changed. It's like this in DefinitelyTyped, so the mistake was made upstream from there. I will have the product team fix that in the source. In the meantime, I think we can live with it in the docs.

File: docs/includes/powerpoint-preview.md (hunk 3 above)
Comment (Rick-Kirkham) [reply docs/includes/powerpoint-preview.md:8]: Hmmm. Existing docs are like this too. I've started an email thread.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**